packages/async-rewriter/src/async-writer-babel.ts

```diff
@@ -526,7 +526,8 @@ var TypeInferenceVisitor: Visitor = { /* eslint no-var:0 */
         if (classPath.node['shellType'].returnType.type === 'unknown') {
           classPath.node['shellType'].returnType.type = 'object';
         }
-        classPath.node['shellType'].returnType.attributes[path.node.key.name] = path.node['shellType'];
+        // TODO: The 'as any' here is probably not really a valid thing to do.
+        classPath.node['shellType'].returnType.attributes[(path.node.key as any).name] = path.node['shellType'];
```

Comment:
I'm surprised that this would change due to dependencies being moved from local to top-level Do you know why it's an issue now and not before?

Reply:
@aherlihy I assume it’s caused by a mismatch in the typescript version that we’re using here